.bazelrc

```diff
@@ -175,11 +175,57 @@ build:remote-ci-linux-local-java8 --host_java_toolchain=@bazel_tools//tools/jdk:
 build:remote-ci-linux-local-java8 --java_toolchain=@bazel_tools//tools/jdk:toolchain_java8
 build:remote-ci-linux-local-java8 --config=remote-ci-common
 #############################################################################
+# remote-ci-linux-coverage: These options are Linux-only using Clang and LLVM coverage
+#############################################################################
+# Clang environment variables (keep in sync with coverage.yml)
+build:remote-ci-linux-coverage --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+build:remote-ci-linux-coverage --action_env=CC=/opt/llvm/bin/clang
+build:remote-ci-linux-coverage --test_env=CC=/opt/llvm/bin/clang
+build:remote-ci-linux-coverage --action_env=CXX=/opt/llvm/bin/clang++
+build:remote-ci-linux-coverage --test_env=CXX=/opt/llvm/bin/clang++
+# Coverage environment variables (keep in sync with coverage.yml)
+build:remote-ci-linux-coverage --action_env=GCOV=/opt/llvm/bin/llvm-profdata
+build:remote-ci-linux-coverage --test_env=GCOV=/opt/llvm/bin/llvm-profdata
+build:remote-ci-linux-coverage --action_env=BAZEL_LLVM_COV=/opt/llvm/bin/llvm-cov
+build:remote-ci-linux-coverage --test_env=BAZEL_LLVM_COV=/opt/llvm/bin/llvm-cov
+build:remote-ci-linux-coverage --action_env=BAZEL_USE_LLVM_NATIVE_COVERAGE=1
+build:remote-ci-linux-coverage --test_env=BAZEL_USE_LLVM_NATIVE_COVERAGE=1
+# Toolchain flags (Java is required for C++ coverage due to Bazel's LCOV merger)
+build:remote-ci-linux-coverage --crosstool_top=@engflow_remote_config_clang_coverage//cc:toolchain
+build:remote-ci-linux-coverage --extra_execution_platforms=@engflow_remote_config_clang_coverage//config:platform
+build:remote-ci-linux-coverage --extra_toolchains=@engflow_remote_config_clang_coverage//config:cc-toolchain
+build:remote-ci-linux-coverage --host_platform=@engflow_remote_config_clang_coverage//config:platform
+build:remote-ci-linux-coverage --platforms=@engflow_remote_config_clang_coverage//config:platform
+build:remote-ci-linux-coverage --extra_toolchains=@local_jdk//:all
+build:remote-ci-linux-coverage --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_java11
+build:remote-ci-linux-coverage --java_toolchain=@bazel_tools//tools/jdk:toolchain_java11
+# Flags to run tests locally which are necessary since Bazel C++ LLVM coverage isn't fully supported for remote builds
+# TODO(lfpino): Reference upstream Bazel issue here on the incompatibility of remote test execution and LLVM coverage.
+build:remote-ci-linux-coverage --test_strategy=standalone
+build:remote-ci-linux-coverage --remote_download_outputs=all
+build:remote-ci-linux-coverage --strategy=TestRunner=local,remote
+build:remote-ci-linux-coverage --strategy=CoverageReport=local,remote
+# Bazel remote caching is incompatible with C++ LLVM coverage so we need to deactivate it for coverage builds
+# TODO(lfpino): Reference upstream Bazel issue here on the incompatibility of remote caching and LLVM coverage.
```

Comment:
Can you repro with a small example?

Reply:
That was my plan when filing the upstream issue, haven't had time to do it just yet.